packages/chat/.storybook/assets/fonts/fonts.css

```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@450&display=swap');
```

Comment:
And not sure if this approach to the font-family is the best @effervescentia

Reply:
I think this should be okay for storybook